app/src/main/java/org/fossasia/susi/ai/device/deviceconnect/DeviceConnectFragment.kt

```diff
@@ -343,40 +503,21 @@ class DeviceConnectFragment : Fragment(), IDeviceConnectView {
     }
 
     override fun setupWiFiAdapter(scanList: ArrayList<String>) {
-        Timber.d("Setup Wifi adapter")
-        scanDevice.setText(R.string.choose_wifi)
-        scanList.remove("SUSI.AI")
-        scanProgress.visibility = View.GONE
-        scanHelp.visibility = View.VISIBLE
-        deviceList.visibility = View.GONE
-        addDeviceButton.visibility = View.GONE
-        wifiList.visibility = View.VISIBLE
-        wifiList.layoutManager = LinearLayoutManager(context)
-        wifiList.setHasFixedSize(true)
-        recyclerAdapter = DevicesAdapter(scanList, deviceConnectPresenter, VIEW_AVAILABLE_WIFI)
-        wifiList.adapter = recyclerAdapter
-    }
-
-    override fun showPopUpDialog() {
-        val utilModel = UtilModel(activity as DeviceActivity)
-        val view = LayoutInflater.from(activity).inflate(R.layout.get_password, null)
-        val alertDialog = AlertDialog.Builder(activity as DeviceActivity).create()
-        alertDialog.setTitle(utilModel.getString(R.string.thanks_wifi))
-        alertDialog.setMessage(utilModel.getString(R.string.enter_password_mail))
-        alertDialog.setCancelable(false)
-
-        val password = view.findViewById<EditText>(R.id.edt_pass)
-
-        alertDialog.setButton(AlertDialog.BUTTON_POSITIVE, utilModel.getString(R.string.next)) { dialog, which ->
-            deviceConnectPresenter.makeAuthRequest(password.text.toString())
+        if (showWifiList) {
+            Timber.e("Setup Wifi adapter")
+            scanDevice.setText(R.string.choose_wifi)
+            scanList.remove("SUSI.AI")
```

Comment:
Get resources, the string is already there.

Reply:
It's not present there. But added it just now

Reply:
You can get the app name also.

Reply:
I think it would be better if we create a new string non-translatable literal and use it. We will be using this literal many times in the future.